src/diffusers/models/transformers/transformer_2d.py

```diff
@@ -100,6 +100,7 @@ def __init__(
         attention_type: str = "default",
         caption_channels: int = None,
         interpolation_scale: float = None,
+        use_additional_conditions: bool = True,
```

Comment:
Should this be defaulted to False?

Reply:
Then how to change these lines to make it correct.
https://github.com/huggingface/diffusers/pull/7654/files/5cbf701dea88b04dd9c859ec015217d74cf19ecd#r1562350687

Reply:
Okay makes sense.